MacPass/Base.lproj/GeneralPreferences.xib

```diff
@@ -19,13 +19,14 @@
                 <outlet property="preventUniversalClipboardSupportCheckButton" destination="nqZ-rB-mFS" id="sbx-rl-reT"/>
                 <outlet property="rememberKeyFileCheckButton" destination="bSt-Wf-FNZ" id="aQm-EA-yAN"/>
                 <outlet property="reopenLastDatabase" destination="530" id="878"/>
+                <outlet property="faviconDownloadMethodPopup" destination="OfU-6f-oTU" id="OfU-6f-oTU-outlet"/>
```

Comment:
Not sure how to generate truly random id here, so I just created derivative one

Reply:
You do not need to adjust anything on those ids. Just let IB do it’s thing and do not mess with the .xib file.